Automattic/jetpack
# Jetpack Social: Turn Social Image Generator off by default

## Proposed changes:
This is a very minor change that turns off Social Image Generator by default, preventing sites from automatically generating social images when they purchase the Advanced plan or otherwise enable Social Image Generator.

### Other information:


## Jetpack product discussion


1202583289493814-as-1204411532038755/f

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Connect your site to Jetpack Social.
* Enable Social Image Generator.
* Go to the Jetpack Social admin page and make sure the toggle is off.
* Create a new post, check that the SIG sidebar is not there, and save the post.
* Check that the post did not save any SIG-related meta: `jetpack docker wp post meta list POST_ID`
* Go back to the Social admin page and enable SIG.
* Create another post, check that the sidebar is there.
* Save the post and check that it has SIG-related meta.
